pom.xml

```diff
@@ -37,6 +37,12 @@
 			<artifactId>jackson-databind</artifactId>
 			<version>2.2.3</version>
 		</dependency>
+		
+		<dependency>
+			<groupId>joda-time</groupId>
+			<artifactId>joda-time</artifactId>
+			<version>2.8.2</version>
+		</dependency>
```

Comment:
My use case is simple for now here so i may not be taking the advantage it has to offer. But in future i have to programmatically find previous terms/term based on the current date, for that i want to try Joda- time so i implemented it now